stdlib/3/multiprocessing/__init__.pyi

```diff
@@ -100,6 +100,7 @@ class Queue():
     def cancel_join_thread(self) -> None: ...
 
 class Value():
+    value: Any = ...
```

Comment:
This is good. The underlying implementation is actually ctypes and each of those types has a `.value` attribute.